src/app/editor.js

```diff
@@ -23,7 +23,9 @@ var css = csjs`
     width             : 100%;
   }
 `
-document.querySelector('#editor-container').className = css['editor-container']
+setTimeout(_ => {
```

Comment:
ah I see. 
I think we should really get rid of the use of `setTimeout` in this way even if this temporary. 
in `index.js`, 
we could have
```
document.body.appendChild(app.render())
app.init()
```
Refactoring by moving code from app.js to each component is for sure something to do but in a long run.
I would for now stick PRs to make room for the dom-console and not refactor the entire app.js .